stored_procedures/safe_crc32_uuid.sql

```diff
@@ -0,0 +1,89 @@
+/*
+Calculate the CRC-32 hash of a UUID using a stored procedure.
+
+Based on https://stackoverflow.com/a/18639999/1260237
+See https://en.wikipedia.org/wiki/Cyclic_redundancy_check
+*/
+CREATE
+OR REPLACE PROCEDURE udf_safe_crc32_uuid(value BYTES, OUT crc INT64) BEGIN DECLARE len INT64;
```

Comment:
please format with the updates in #693, and add this file to `SKIP` in [script/format_sql](https://github.com/mozilla/bigquery-etl/blob/master/script/format_sql#L17)